src/main/java/org/zone/commands/structure/region/flags/display/leaving/ZoneFlagLeavingMessageDisplaySetBossBarCommand.java

```diff
@@ -26,6 +30,11 @@
     public static final ZoneArgument ZONE_ID = new ZoneArgument("zoneId",
             new ZoneArgument.ZoneArgumentPropertiesBuilder().setBypassSuggestionPermission(
                     ZonePermissions.OVERRIDE_FLAG_LEAVING_MESSAGE_SET_BOSS_BAR));
+    public static final IntegerArgument PROGRESS = new IntegerArgument("progress");
```

Comment:
This is probably better as a ``RangeArgument``

Reply:
k